tests/Tests.cs

```diff
@@ -83,6 +83,66 @@ public class ControllerTests
 		}
 
 		[Fact]
+		public async void valid_products_by_single_tag()
```

Comment:
Add `_from_real_api` to the end of the name of this and the other test so it is clear that these tests don't use fake data.